ImGuiFileDialog.h

```diff
@@ -16,40 +21,42 @@ struct FileInfoStruct
 
 class ImGuiFileDialog
 {
-private:
```

Comment:
Feel free to ignore my changes in this file. Mostly just grouping all public stuff together and all private stuff together rather than have multiples of each. The original code had three successive public: declarations. The subsequent two are redundant.

I also re-ordered the members according to my personal preference (and the most common way of doing it).